libbeat/template/load.go

```diff
@@ -124,11 +125,11 @@ func (l *ESLoader) templateExists(templateName string) bool {
 	if l.client == nil {
 		return false
 	}
-	status, _, _ := l.client.Request("HEAD", "/_template/"+templateName, "", nil, nil)
-	if status != http.StatusOK {
-		return false
+	status, body, _ := l.client.Request("GET", "/_cat/templates/"+templateName, "", nil, nil)
+	if status == http.StatusOK && strings.Contains(string(body), templateName) {
+		return true
 	}
-	return true
+	return false
```

Comment:
nit (I know it's been this way before): `return <predicate>` instead of `if <predicate> { return true }; return false`.